Source/Core/DiscIO/DirectoryBlob.cpp

```diff
@@ -350,10 +350,21 @@ static bool IsBootBin(const std::string& path)
   return File::Exists(partition_root + "sys/main.dol");
 }
 
+static bool IsHeaderBin(const std::string& path)
+{
+  if (!PathEndsWith(path, "/disc/header.bin"))
```

Comment:
nitpick: If you establish a `constexpr` for "/disc/header.bin" then it wouldn't have to be typed out twice.

Reply:
It would be more typing overall, and also the second string is lacking the leading slash.

Reply:
It doesn't need to lack the leading slash if you added it back with "/sys/main.dol". ;)
It's fine as-is. I just prefer to establish a variable name for the string to establish a connection between multiple usages, not that this particular situation is overly confusing.